cmserver/cmserver.go

```diff
@@ -63,12 +64,28 @@ type UpdateStatus struct {
 	Error string
 }
 
+// UpdateFOTAStatus struct with fota parts information
```

Comment:
Change comment: `FOTA update status for update scheduler service` or something similar.